dashboard/src/utils/helper.js

```diff
@@ -5,6 +5,13 @@ export const uid = () => {
   return head + tail;
 };
 
+/**
+ * Check if password entered by user is valid or not
+ * @function
+ * @param {string} password - Entered email
+ * @param {number} passwordLength - Length of Password
+ * @returns {boolean} - Password valid or not
```

Comment:
For boolean returns, it's good to be more specific about what the values mean. "`true` if the password is valid", for example.